lib/temporal/SQL/update_str3ds_metadata_template.sql

```diff
@@ -34,6 +34,6 @@ UPDATE str3ds_metadata
            max(ewres) AS ewres_max_new,
            min(tbres) AS tbres_min_new,
            max(tbres) AS tbres_max_new
-       FROM SPACETIME_REGISTER_TABLE INNER JOIN raster3d_metadata USING (id)
+       FROM SPACETIME_REGISTER_TABLE NATURAL INNER JOIN raster3d_metadata
```

Comment:
Is there a good reason to use NATURAL joins? Usually they are never used in production code. It is not predictable or explicit enough. If one wants an inner join, he writes it, if its an outer join, he writes it, and the better you can write out the constraints, the better the query plan.

https://stackoverflow.com/questions/13545476/sqlite-natural-join-broken
https://www.reddit.com/r/SQL/comments/tweggn/natural_join_vs_inner_join_question/
https://stackoverflow.com/questions/44301641/is-natural-join-the-only-elision-of-foreign-key-name
https://stackoverflow.com/questions/36230571/sqlite-query-with-natural-join-produces-extra-records

The other aspect to maybe consider, is that with a little bit less (R)DBMS-specific queries, it's a bit easier to have the same or almost the same queries across drivers.

Reply:
Good points. I switched to `JOIN-ON`. If you have suggestions for #3350 I would be happy to hear, as the performance improvement there is more significant...

Reply:
Thanks! Thats what happens when cross domain knowledge gets useful. I'll take a look at your referenced PR.